app/constants/owner.js

```diff
@@ -32,7 +32,7 @@ const constants = {
     ownerDetails: 'cdo/create/owner-details',
     selectAddress: 'cdo/create/select-address',
     address: 'cdo/create/address',
-    enforcementDetails: 'cdo/create/enforcement-details',
+    enforcementDetails: 'cdo/create/_enforcement-details',
```

Comment:
Is there a reason for the underscore here?

Reply:
I have followed the convention of the dynamic views (generated from templates) where they use an underscore (e.g. _layout.template.njk generates _layout.njk, and _cookie-banner.template.njk generates _cookie-banner.njk). The generated njk files don't get checked in (added to gitignore) but get generated whenever the code is built. The dynamic generation is where webpack files get dynamically referenced in the njk